GitUI/CommandsDialogs/RevisionFileTreeControl.cs

```diff
@@ -404,9 +418,8 @@ private Task ShowGitItemAsync(GitItem gitItem)
                             return ViewGitItemAsync(gitItem);
                         }
 
-                        int? line = FileText.Visible ? FileText.CurrentFileLine : null;
-                        BlameControl.Visible = true;
                         FileText.Visible = false;
+                        BlameControl.Visible = true;
```

Comment:
This makes the FileStatusList's filter field get the focus when hitting `B` multiple times (with `Show blame in diff viewer` on).

Reply:
That is different controls, how can that be related?

Reply:
Not directly, you are right. I hoped that BlameControl would be focused automatically but it isn't.

STR:
- focus FileText
- hit `B` twice

So, BlameControl should be focused explicitly if FileText was focused.

Reply:
This behavior is not changed by this PR, it is a separate issue. (RevDiff is not changed here other than setting line if Blame is done in FileTree).
I have never the less not been able to find a workaround but will continue to look at it in #9804.